packages/flutter/lib/src/services/text_input.dart

```diff
@@ -407,6 +407,72 @@ enum TextCapitalization {
   none,
 }
 
+/// Configures how the platform will offer auto-fill suggestions for the input
+enum TextContentType {
```

Comment:
Both [UITextContentType](https://developer.apple.com/documentation/uikit/uitextcontenttype) and Android autofill hints are raw strings under the hood, and you can create them using arbitrary strings. `enum` is a closed type. Should we just use `String` instead or use some wrapper class that allows for custom content types?

Reply:
👍 I like the idea of using something closer to the raw value. That would make it easier to maintain as they continue to add content types.

Reply:
@LongCatIsLooong I've updated that enum to a custom class. Let me know if this is what you had in mind.